widgetssdk/src/main/java/com/glia/widgets/chat/ChatActivity.java

```diff
@@ -52,13 +53,13 @@ public final class ChatActivity extends FadeTransitionActivity {
      * Creates and fills out Intent for starting ChatActivity
      * @param context   - Context object
      * @param contextId - Context asset ID
-     * @param queueId   - Queue ID
+     * @param queueId   - Queue ID or `null` to request the default queues
```

Comment:
NIT: I would change **request** to **use**
```suggestion
     * @param queueId   - Queue ID or `null` to use the default queues
```

Reply:
Same in other places